js/localization/globalize/date.js

```diff
@@ -114,8 +114,8 @@ if(Globalize && Globalize.formatDate) {
             return Globalize.locale().main('dates/calendars/gregorian/' + path);
         },
 
-        getPeriodNames: function() {
```

Comment:
please, add tests for datebox localization. U can find it in this PR: https://github.com/DevExpress/DevExtreme/pull/24451

Reply:
added